reladomo/src/main/java/com/gs/fw/common/mithra/attribute/ByteAttribute.java

```diff
@@ -537,7 +537,7 @@ public BigDecimalAttribute dividedBy(BigDecimalAttribute attribute)
     @Override
     public ByteAttribute absoluteValue()
     {
-        throw new UnsupportedOperationException("absoluteValue is not implemented for ShortAttribute");
+        throw new UnsupportedOperationException("absoluteValue is not implemented for ByteAttribute");

```

Comment:
This is unrelated but I found it when adding test coverage for absoluteValue(). I did this first to make sure I understood derived attributes.